src/index.js

```diff
@@ -37,7 +37,7 @@ const INCLUDE_RENDER_OPTS = {
 let msg = act => `expected #{act} to ${act} #{exp}`;
 
 // assert that an object is JSX (or more correctly, a VNode)
-let isJsx = obj => obj && (options.isJsx ? options.isJsx(obj) : (obj.__isVNode || isVNode(obj)));
+let isJsx = obj => obj && (options.isJsx ? options.isJsx(obj) : (obj._vnode || isVNode(obj)));
```

Comment:
I'm going to merge this, but @pl12133 I can't seem to remember what `__isVNode` was being used for. It doesn't seem to have ever been in preact or compat. Do you know?

Reply:
I went through some history and it seems like this was in `preact` since the beginning, but it must have been removed a long time ago:

https://github.com/preactjs/preact/blob/e9f1a5645ef7d10ba05c601b94d70e4e4d0bd57f/src/preact.js#L204